analyses/tumor-purity-exploration/01_explore-tumor-purity.Rmd

```diff
@@ -0,0 +1,259 @@
+         cancer_group_display = forcats::fct_relevel(cancer_group_display, "Other", after = Inf),
+         broad_histology_display = forcats::fct_reorder(broad_histology_display, tumor_fraction), 
+         broad_histology_display = forcats::fct_relevel(broad_histology_display, "Other", after = Inf))
+         
+# Cancer group
+plot_tumor_violins(tumor_palette_df, cancer_group_display, cancer_group_hex)
+
+# Broad histology
+plot_tumor_violins(tumor_palette_df, broad_histology_display, broad_histology_hex)
+```
+
+
+
+
+Distribution across sequencing centers, highlighting cancer groups:
+
+```{r}
+# use jitter to highlight diagnoses
+seq_center_df <- tumor_palette_df %>%
+  inner_join(
+    select(metadata, 
+           Kids_First_Biospecimen_ID, 
+           seq_center)
+  )
+
+ggplot(seq_center_df) +
+  aes(x = seq_center, 
+      y = tumor_fraction, 
+      color = cancer_group_hex) + 
+  geom_jitter() + 
+  scale_color_identity() +
+  theme(axis.text.x = element_text(angle = 30, hjust=1))
+
+# What are the cancer groups at the smaller centers?
+seq_center_df %>%
+  count(seq_center, cancer_group_display) %>%
+  arrange(seq_center)
+```
+
+
+## Tumor fraction at the RNA level
+
+Tumor fractions explored above were derived from WGS data.
+Estimating tumor fraction from RNA directly is challenging since methods focus on estimating proportion of immune cells and assuming the rest is tumor, which is not a fair assumption for many of these diagnoses.
+
+Therefore, for RNA data tumor purity estimates, we should instead directly use tumor fraction values but _only for those samples where DNA and RNA sequencing were performed on the same extraction_.
+We cannot really know the RNA tumor purity for samples that have separate DNA/RNA extractions.
+
+This extraction information is present in data release `>=v23` - we're interested in "Same extraction" samples.
+First, how many such samples are there?
+
+```{r}
+table(metadata$extraction_type)
+```
+We'll need to get the RNA equivalents (since these are all WGS), and we'll map over this information using the `sample_id` information, as was done for the oncoprint module:
+https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/5fe10b88aaa5176fa6577ffd6c8388be05098e83/analyses/oncoprint-landscape/01-map-to-sample_id.R
+
+
+As part of this, we want to find the ambiguous `sample_id` samples so that we don't include them, as described:
+https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/5fe10b88aaa5176fa6577ffd6c8388be05098e83/analyses/oncoprint-landscape/01-map-to-sample_id.R#L153
+
+> An ambiguous sample_id will have more than 2 rows associated with it in the histologies file when looking at tumor samples -- that means we won't be able to determine when an WGS/WXS assay maps to an RNA-seq assay
+
+
+```{r}
+# find the ambiguous samples to flag for removal
+# here the composition does not matter, unlike in the linked oncoprint filtering
```

Comment:
Hmm... I'm not sure I agree with this. Not necessarily here... but when we look at tumor purity above, it strikes me as a little bit strange to include cell lines? I think this comment may be more important for the next notebook with thresholds or when we think about making plots using the thresholds.

However, it seems that it won't matter in practice because only one cell line is going to pass the same extraction filtering step.

Reply:
Update: briefly looked at the _TP53_ classifier and EXTEND modules (analysis modules, didn't look at figures), I think we lump in cell lines to those analyses so 🤷🏻 

Maybe the composition included in plots should be made more clear like the fact that we always use the standed data.